### PR TITLE
Login logic

### PR DESCRIPTION
**Pull request by kaylebredner**

### CODE REVIEW

File: server.js
Comment (tarunJeevan) [inline server.js:23]: Have you looked up any videos or guides on making multi-user authentication?

File: server.js
Comment (tarunJeevan) [inline server.js:24]: Is this variable supposed to be the same one from the model? Because the model says 'authenticated' must be of type Number

File: server.js
Comment (tarunJeevan) [inline server.js:70]: Don't use var. Use let if you need to. Var's scope is the entire program, so use let instead.

File: server.js
Comment (tarunJeevan) [inline server.js:63]: Again, you're creating userName and pw in this method, and you never reassign them, so using const is the safer option

**Review by tarunJeevan — COMMENTED** [review]
Looks pretty good overall, but I found a few things like using var or let instead of const and other things like that. Can you change those things here or do you plan on doing that after you merge into main?
I'm selecting Comment for now, but I'll Approve it if you want me to.

File: server.js
Comment (kaylebredner) [reply server.js:24]: going to do that with the database, I wrote a function for it

File: server.js
Comment (kaylebredner) [reply server.js:41]: I think if I change that to const it won't let me assign the fields in the object, could be wrong though

File: server.js
Comment (kaylebredner) [reply server.js:25]: That variable won't be used after authentication is reworked

**Review by maitmd — APPROVED** [review]
This looks good!

**Review by AlexBishop01 — APPROVED** [review]

**Review by tarunJeevan — APPROVED** [review]

**Pull request merged**
